.gitignore

```diff
@@ -33,4 +33,4 @@ bin/**
 # *.pytest
 
 .idea/
-TEST/
+/TEST/
```

Comment:
Can you leave the other one (either `TEST/` or `./TEST/` in place please?

Reply:
I cannot. My Windows filesystem is case insensitive, and so this line ignores `src/test`

Reply:
and `./TEST/` does not ignore the TEST directory :/

Reply:
@voj no you're right the `/` here refers to git project root, and NOT the FileSystem root. So it works as it should for me too - thanks